app/Actions/Diagnostics/Checks/PHPVersionCheck.php

```diff
@@ -49,7 +44,7 @@ public function check(array &$errors): void
 		}
 
 		// Extensions
-		$extensions = ['session', 'exif', 'mbstring', 'gd', 'PDO', 'json', 'zip', 'intl'];
+		$extensions = ['session', 'exif', 'mbstring', 'gd', 'PDO', 'json', 'zip'];
```

Comment:
Why has this become unnecessary?

Reply:
it was removed from the Laravel required package for installation.